helper/testhelpers/corehelpers/corehelpers.go

```diff
@@ -232,7 +232,7 @@ func TestNoopAudit(t testing.T, path string, config map[string]string, opts ...a
 // NewNoopAudit should be used to create a NoopAudit as it handles creation of a
 // predictable salt and wraps eventlogger nodes so information can be retrieved on
 // what they've seen or formatted.
-func NewNoopAudit(config *audit.BackendConfig, opts ...audit.Option) (*NoopAudit, error) {
+func NewNoopAudit(config *audit.BackendConfig, headerFormatter audit.HeaderFormatter) (*NoopAudit, error) {
```

Comment:
Has there been any thought into defining a NoopHeaderFormatter and simply implying that here rather than taking an **audit.HeaderFormatter** as a parameter? It could replace all of those repeated instances of testHeaderFormatter that appear in the various _builtin/audit/..._ sub-packages.